Resources/doc/reference/filter_field_definition.rst

```diff
@@ -179,3 +179,31 @@ implement this functionality.
             return true;
         }
     }
+
+Filtering case insensitive strings
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Normally the query builder behind the filtering would compare field values case sensitive. There can be reasons to
+filter case insensitive. To do so you can use the ``compare_case_insensitive`` option to force a case insensitive
+comparison on field value and filter value.
+
+.. code-block:: php
+
+    <?php
+    namespace My\AppBundle\Admin;
+
+    use Sonata\AdminBundle\Admin\Admin;
+    use Sonata\AdminBundle\Datagrid\DatagridMapper;
+
+    use Application\Sonata\NewsBundle\Document\Comment;
+
+    class PostAdmin extends Admin
+    {
+        protected function configureDatagridFilters(DatagridMapper $datagridMapper)
+        {
+            $datagridMapper
+                ->add('title')
+                ->add('author')
+                ->add('with_case_insensitive_comparison', 'doctrine_phpcr_string', array('compare_case_insensitive' => true))
```

Comment:
i would make the title and author fields case insensitive filtering to make the example more real life